src/webview/webViewEventHandlers.js

```diff
@@ -270,6 +276,18 @@ export const handleMessageListEvent = (props: Props, _: GetText, event: MessageL
       console.debug(props, event); // eslint-disable-line
       break;
 
+    case 'time': {
```

Comment:
Let's not break up `debug` from `warn` and `error` -- as is, they form a nice sequence of closely related cases.

Reply:
Oh yes! Sorry about this :(